libkbfs/interfaces.go

```diff
@@ -28,6 +28,10 @@ type logMaker interface {
 	MakeLogger(module string) logger.Logger
 }
 
+type debugForcedLogMaker interface {
```

Comment:
This doesn't seem to be used anywhere, probably doesn't need to be a new interface.

Reply:
Sure; I'll put it into `logMaker` then.

Reply:
Ah it looks like `testLogMaker` maps to `logMaker` too. I'll just put it in `Config` I guess.